src/process_utility.c

```diff
@@ -1478,13 +1478,22 @@ process_reindex(ProcessUtilityArgs *args)
 			{
 				PreventCommandDuringRecovery("REINDEX");
 				ts_hypertable_permissions_check_by_id(ht->fd.id);
-
+#if PG12_GE
+				if (stmt->concurrent)
+				{
+					stmt->concurrent = false;
+					ereport(NOTICE,
+							(errmsg("REINDEX TABLE CONCURRENTLY is unsupported on a hypertable"),
+							 errdetail("Using non-concurrent implementation.")));
```

Comment:
Again, not sure we need this since we should probably just generate an error above.

```suggestion
							 errdetail("Performing non-concurrent reindexing instead.")));
```